oak_crypto/src/encryptor.rs

```diff
@@ -104,6 +104,30 @@ impl EncryptionKeyProvider {
     }
 }
 
+// This trait just aliases the `RecipientContextGenerator`, while using different naming
+// as defined in the Oak SDK design doc.
+// TODO(#3841): rename the relevant trait and struct in our crypto crates.
+/// Generate [`SessionKeys`] for the provided public key.
+pub trait EncryptionKeyHandle {
+    fn generate_session_keys(&self, encapsulated_public_key: &[u8]) -> anyhow::Result<SessionKeys>;
+}
+
+// Alias this struct in order to conform to the naming outlined in the restricted kernel SDK design
+// doc.
+use crate::encryptor::RecipientContext as SessionKeys;
```

Comment:
Is this just temporary in order to do the renaming without breaking callers?

Reply:
yup :)

Reply:
added to do to make that clear